packages/pn-pa-webapp/src/pages/__test__/Statistics.page.test.tsx

```diff
@@ -198,9 +197,7 @@ describe('Statistics Page tests', () => {
       expect(mock.history.get[0].url).toBe(
         `/bff/v1/sender-dashboard/dashboard-data-request/PA/${
           userResponse.organization.id
-        }?startDate=${formatToSlicedISOString(twelveMonthsAgo)}&endDate=${formatToSlicedISOString(
-          today
-        )}`
+        }?startDate=${formatToSlicedISOString(twelveMonthsAgo)}&endDate=${rawResponseMock.lastDate}`
```

Comment:
this is strange because I would expect that the first api call is done with endDate equal to today